library/bignum_core.h

```diff
@@ -452,4 +452,23 @@ void mbedtls_mpi_core_montmul( mbedtls_mpi_uint *X,
 int mbedtls_mpi_core_get_mont_r2_unsafe( mbedtls_mpi *X,
                                          const mbedtls_mpi *N );
 
+/**
+ * Select an MPI from a table without leaking the index.
+ *
+ * \param dest              The destination buffer. This must point to a writable
+ *                          buffer of at least \p limbs limbs.
+ * \param table             The address of the table. This must point to a readable
+ *                          array of \p count elements of
+ *                          \p limbs limbs each each.
+ * \param limbs             The length of a table entry in limbs.
+ * \param count             The number of elements in \p table.
+ * \param index             The secret table index to look up. This must be in the
+ *                          range `0,..,count-1`.
+ */
+void mbedtls_mpi_core_ct_uint_table_lookup( mbedtls_mpi_uint *dest,
```

Comment:
Should that be *X following the convention agreed on previous reviews?

Reply:
This isn't an arithmetic function, hence why I didn't query this. I'd say leave it as `dest`